packages/core/src/components/Table/TablePaginator.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { View, Text } from "react-native";
+
+const TablePaginator = () => (
```

Comment:
Is this file needed? I dont get what it does.

Reply:
I had created this file initially thinking we might support pagination but we decided to go without pagination that time. I can remove this file.